apps/portal/src/content/docs/components/link.mdx

```diff
@@ -0,0 +1,180 @@
+    </div>
+
+    <div className="flex flex-col gap-2">
+        <h5 className="mb-1">Default w/prefix icon</h5>
+        <Link prefixIcon href="">Link</Link>
+        <Link prefixIcon href="" data-hovered="true">Link (hovered)</Link>
+        <Link prefixIcon href="" disabled>Link (disabled)</Link>
+    </div>
+
+    <div className="flex flex-col gap-2">
+        <h5 className="mb-1">Default sm w/prefix icon</h5>
+        <Link size="sm" prefixIcon href="">Link</Link>
+        <Link size="sm" prefixIcon href="" data-hovered="true">Link (hovered)</Link>
+        <Link size="sm" prefixIcon href="" disabled>Link (disabled)</Link>
+    </div>
+
+    <div className="flex flex-col gap-2">
+        <h5 className="mb-1">Secondary w/prefix icon</h5>
+        <Link prefixIcon variant="secondary" href="">Link</Link>
+        <Link prefixIcon variant="secondary" href="" data-hovered="true">Link (hovered)</Link>
+        <Link prefixIcon variant="secondary" href="" disabled>Link (disabled)</Link>
+    </div>
+
+    <div className="flex flex-col gap-2">
+        <h5 className="mb-1">Secondary sm w/prefix icon</h5>
+        <Link size="sm" prefixIcon variant="secondary" href="">Link</Link>
+        <Link size="sm" prefixIcon variant="secondary" href="" data-hovered="true">Link (hovered)</Link>
+        <Link size="sm" prefixIcon variant="secondary" href="" disabled>Link (disabled)</Link>
+    </div>
+
+    <div className="flex flex-col gap-2">
+        <h5 className="mb-1">Default w/suffix icon</h5>
+        <Link suffixIcon href="">Link</Link>
+        <Link suffixIcon href="" data-hovered="true">Link (hovered)</Link>
+        <Link suffixIcon href="" disabled>Link (disabled)</Link>
+    </div>
+
+    <div className="flex flex-col gap-2">
+        <h5 className="mb-1">Default sm w/suffix icon</h5>
+        <Link size="sm" suffixIcon href="">Link</Link>
+        <Link size="sm" suffixIcon href="" data-hovered="true">Link (hovered)</Link>
+        <Link size="sm" suffixIcon href="" disabled>Link (disabled)</Link>
+    </div>
+
+    <div className="flex flex-col gap-2">
+        <h5 className="mb-1">Secondary w/suffix icon</h5>
+        <Link suffixIcon variant="secondary" href="">Link</Link>
+        <Link suffixIcon variant="secondary" href="" data-hovered="true">Link (hovered)</Link>
+        <Link suffixIcon variant="secondary" href="" disabled>Link (disabled)</Link>
+    </div>
+
+    <div className="flex flex-col gap-2">
+        <h5 className="mb-1">Secondary sm w/suffix icon</h5>
+        <Link size="sm" suffixIcon variant="secondary" href="">Link</Link>
+        <Link size="sm" suffixIcon variant="secondary" href="" data-hovered="true">Link (hovered)</Link>
+        <Link size="sm" suffixIcon variant="secondary" href="" disabled>Link (disabled)</Link>
+    </div>
+
+</div>`}
+/>
+
+## Usage
+
+```typescript jsx
+import { Link } from '@harnessio/ui/components'
+
+// ...
+
+return (
+  <>
+    <Link href="/your-url-here">Username</Link>
+  </>
+)
```

Comment:
You can drop the fragment and go single-line
```suggestion
return <Link href="/your-url-here">Username</Link>
```

Reply:
Fixed